docs/index.md

```diff
@@ -43,10 +43,11 @@ To also install our recommended dependencies:
 pip install -U mesa[rec]
 ```
 
+The `[rec]` option installs additional recommended dependencies needed for visualization (SolaraViz), Matplotlib plotting, and network modeling capabilities.
```

Comment:
add altair so `matplotlib and altair plotting`

Reply:
another question: do we want to name the libraries explicitly and link them to the functionality for which they are used?

Reply:
Let’s just remove the specific libraries:
```suggestion
The `[rec]` option installs additional recommended dependencies needed for visualization, plotting, and network modeling capabilities.
```